### PR TITLE
fix: mypy errors in infra, runtime and value_objects

### PR DESCRIPTION
**Pull request by hsgwa**
This PR fixes mypy errors which are easy to fix.
plot-related errors and records-related errors are still remain.

I'm gonna to enable mypy-tests in Github actions for code quality.

### CODE REVIEW

File: src/caret_analyze/value_objects/message_context.py [hunk 1]
```diff
@@ -259,7 +259,7 @@ def verify(self) -> bool:
             is_valid = False
 
             # Check binding between callback and publisher
-            if not self._pub.summary['callbacks']:
+            if self._pub and not self._pub.summary['callbacks']:
```
Comment (keita1523) [inline src/caret_analyze/value_objects/message_context.py:262]: Why did you add this code?

Comment (hsgwa) [reply src/caret_analyze/value_objects/message_context.py:262]: > For adapting mypy error?

Yes.
self._pub is `Optional[PublisherStructValue]`.
defined here: [For adapting mypy error?](https://github.com/tier4/CARET_analyze/pull/147/files/9f772362b578bc24e949767d354b96d29a7b4a64#diff-042e0f9d1a23e65575b6056c3f42c4bdb046cee2763f22775d868aa83e54e73eR71)


This modification fixes an error when self._pub is None.

Comment (hsgwa) [reply src/caret_analyze/value_objects/message_context.py]: I modified self._pub is None case: 2bd9b9b

**Comment by keita1523** [comment]
LGTM

**Comment by keita1523** [comment]
@hsgwa 
https://github.com/tier4/CARET_analyze/actions/runs/3126685139/jobs/5072466354

Pytest is fail.
Is it OK to ignore it?

**Comment by hsgwa** [comment]
@keita1523 
fixed in : ed0e397

> Pytest is fail.
Is it OK to ignore it?

**Review by keita1523 — APPROVED** [review]
LGTM

**Review by takeshi-iwanari — APPROVED** [review]
LGTM
